disasm-test/Main.hs

```diff
@@ -493,9 +550,30 @@ dropTrailingSpace bs
 -- | Ignore a command that fails.
 ignore :: IO () -> IO ()
 ignore  = X.handle f
-  where f   :: X.IOException -> IO ()
+  where f   :: EX.IOException -> IO ()
         f _ = return ()
 
-callProc :: String -> [String] -> IO ()
-callProc p args = -- putStrLn ("Calling process: " ++ p ++ " " ++ unwords args) >>
-  Proc.callProcess p args
+callProc :: String -> [String] -> TestMonad ()
+callProc p args = do
+  Details dets <- gets showDetails
+  when dets $ liftIO $ putStrLn ("## Running: " ++ p ++ " " ++ unwords args)
+  liftIO $ Proc.callProcess p args
+
+withFile :: TestMonad FilePath -> (FilePath -> TestMonad r) -> TestMonad r
+withFile iofile f = X.bracket iofile rmFile f
+
+with2Files :: TestMonad (FilePath, Maybe FilePath)
```

Comment:
"`with2Files`" is an overly broad name for something that is really quite specialized. In fact, `with2Files` is only ever called in applied to `processLL` as its first argument, so I'd even propose combining `with2Files`' functionality into `processLL`.

Reply:
There will be a subsequent PR that will utilized `with2Files` from a different location, so I'd like to keep it separated.  I'm willing to rename it if you have suggestions; I agree it's not a great name, but this is just within this application and not a library API, so I was a little more accepting of imprecision.

Reply:
Ah, I didn't realize this was destined for other uses. In that case, I'm fine with this (I don't have a better name suggestion).